api/tests/acceptance/scripts/bigint/answer/copy-newly-inserted-rows-to-answerId-tmp-bigint-tables_test.js

```diff
@@ -0,0 +1,88 @@
+const {
+  createAnswersBigintMigrationDatabaseStructures,
+} = require('../../../../../scripts/bigint/answer/create-migration-database-structure');
+const {
+  copyAnswerKeDataToTemporaryTables,
+} = require('../../../../../scripts/bigint/answer/copy-answers-ke-data-to-temporary-tables-with-bigint');
+const { initBigintMigrationSetting } = require('../../../../../scripts/bigint/answer/init-bigint-migration-setting');
+const {
+  addConstraintsAndIndexesToAnswersKeBigintTemporaryTables,
+} = require('../../../../../scripts/bigint/answer/add-constraints-and-indexes-to-answers-ke-temporary-tables-with-bigint');
+const {
+  copyNewlyInsertedRowsToAnswerIdTmpBigintTables,
+} = require('../../../../../scripts/bigint/answer/copy-newly-inserted-rows-to-answerId-tmp-bigint-tables');
+
+const { expect, knex } = require('../../../../test-helper');
+
+const DatabaseBuilder = require('../../../../../db/database-builder/database-builder');
+const databaseBuilder = new DatabaseBuilder({ knex });
+
+describe('#copyNewlyInsertedRowsToAnswerIdTmpBigintTables', function () {
+  it('should add triggers into answers and KE bigint tables and copy newly inserted rows', async function () {
+    // given
+    await knex.raw('DROP TABLE IF EXISTS "bigint-migration-settings"');
+    await knex.raw('DROP TABLE IF EXISTS "knowledge-elements_bigint"');
+    await knex.raw('DROP TABLE IF EXISTS "answers_bigint"');
+    await knex.raw('DROP TRIGGER IF EXISTS trg_answers ON answers');
+    await knex.raw('DROP TRIGGER IF EXISTS "trg_knowledge-elements" ON "knowledge-elements"');
+
+    await createAnswersBigintMigrationDatabaseStructures(knex);
+
+    // when
+    const { id: assessmentId } = databaseBuilder.factory.buildAssessment();
+    const { id: firstAnswerId } = databaseBuilder.factory.buildAnswer({ assessmentId });
+    databaseBuilder.factory.buildKnowledgeElement({ assessmentId, answerId: firstAnswerId });
+    const { id: secondAnswerId } = databaseBuilder.factory.buildAnswer({ assessmentId });
+    databaseBuilder.factory.buildKnowledgeElement({ assessmentId, answerId: secondAnswerId });
+
+    await databaseBuilder.commit();
+
+    await copyAnswerKeDataToTemporaryTables();
+    await initBigintMigrationSetting();
+    await addConstraintsAndIndexesToAnswersKeBigintTemporaryTables();
+    await copyNewlyInsertedRowsToAnswerIdTmpBigintTables();
+
+    // then
+    const { rows: triggersByTable } = await knex.raw(
+      `SELECT  event_object_table AS table_name ,trigger_name
+             FROM information_schema.triggers
+             GROUP BY table_name , trigger_name
+             ORDER BY table_name ,trigger_name
+            `
+    );
+    expect(triggersByTable).to.deep.equal([
+      {
+        table_name: 'answers',
+        trigger_name: 'trg_answers',
+      },
+      {
+        table_name: 'knowledge-elements',
+        trigger_name: 'trg_knowledge-elements',
+      },
+    ]);
```

Comment:
Proposition: isoler cette assertion dans un `it` (et mettre le when dans un `beforeEach`)